build.gradle

```diff
@@ -26,7 +26,7 @@ buildscript {
 
         common_utils_version = System.getProperty("common_utils.version", '3.2.0.0-SNAPSHOT')
 
-        kafka_version  = '4.1.0'
+        kafka_version  = '4.0.0'
```

Comment:
This is intentional. Now that the tests are running they are failing with this version bump so https://github.com/opensearch-project/security/pull/5613 needs to be reverted.

I'll take a quick look to see if there's an additional dep that needs to be added.